src/app/Modules/home/profile/profile.component.html

```diff
@@ -12,9 +12,11 @@
             </button>
         </section>
         <div class="profile-section">
-            <div>
-                <img *ngIf="user.urlImg === undefined" class="profile-img" src="../../../assets/images/profileImg/profile-img-0.png" alt="">
-                <img *ngIf="user.urlImg !== undefined" class="profile-img" src="../../../assets/images/profileImg/{{user.urlImg}}" alt="">
+            <div *ngIf="!isedit">
```

Comment:
Can combine this and the following element in one by using the ternary operator on the class and src.